site/content/deployment/resource-limitation.md

```diff
@@ -105,3 +105,5 @@ The `resources` property defines the overall limit of 256 MB of memory that the
 ## Limiting the number of device connections
 Resource limits namely the maximum number of device connections allowed per tenant can be set in Hono. Please refer to the [connections limit concept] ({{< ref "/concepts/resource-limits.md#connections-limit" >}}) for more information. The resource-limits for a tenant can be set using the tenant configuration and please refer to the [Tenant API]({{< relref "/api/Tenant-API.md#request-payload" >}}) for the configuration details.
 
+## Limiting the number of telemetry and event messages
+A limit on the incoming telemetry and event messages per tenant can be set in Hono. Please refer to the [messages limit concept] ({{< ref "/concepts/resource-limits.md#messages-limit" >}}) for more information. The resource-limits for a tenant can be set using the tenant configuration and please refer to the [Tenant API]({{< relref "/api/Tenant-API.md#request-payload" >}}) for the configuration details.
```

Comment:
pleas use a `ref` instead of `relref`. Please also fix this in the previous section ..